docs/mobile-apps/features/virtual-usb.md

```diff
@@ -282,11 +287,11 @@ adb connect localhost:7000
 
 ### Test and Debug
 
-8. Now, you can debug and run tests on your app. For guidance and ideas, see the [Example Use Cases](/mobile-apps/features/virtual-usb/#example-use-cases).
+8. Now, you can debug and run tests on your app. We advise you to use our Sauce App Storage to upload the apps and run tests against them. 
 
 <!-- prettier-ignore-start -->
-:::caution iOS Limitation
-To do proper debugging, the iOS device symbols will need to be downloaded to your local machine. This happens automatically when you're connecting to a Sauce Labs iOS device for the first time via a remote debug vUSB session with Xcode. **Xcode will attempt to download the iOS device symbols over the vUSB tunnel, causing a lag that can last up to a few minutes.**
+:::caution iOS Limitations
+Attaching debugger is not supported, due to the amount of data that the iOS device symbols will need to be downloaded to your local machine. This happens automatically when you're connecting to a Sauce Labs iOS device for the first time via a remote debug vUSB session with Xcode. **Xcode will attempt to download the iOS device symbols over the vUSB tunnel, causing a lag that can last from few minutes up to an hour.**
```

Comment:
Small grammatical changes:
Attaching a debugger is not supported due to the amount of data that the iOS device symbols will need to be downloaded to your local machine. This process occurs automatically when you're connecting to a Sauce Labs iOS device for the first time via a remote debug vUSB session with Xcode. **Xcode will attempt to download the iOS device symbols over the vUSB tunnel, causing a lag that can last from a few minutes to an hour.**